.travis.yml

```diff
@@ -15,6 +15,7 @@ matrix:
         - python3.4
         - python3.5
         - python3.6
+        - python3.7
```

Comment:
It might be failed due to a lack of supporting Python 3.7 on Travis-CI. I bypassed it on [another repository](https://github.com/admire93/pyleftpad/blob/master/.travis.yml#L7-L12).

Reply:
https://github.com/nirum-lang/nirum/blob/bc8fc1a3d6fd25c01a9c565155bb0ab3e7ec9506/.travis.yml#L5-L23

Currently, `.travis.yml` file uses apt with deadsnakes PPA.
So if the deadsnakes PPA brokes then there would be a modify for the proper versions support.